src/Sql/Sql.Test/Sql.Test.csproj

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.22.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.24.0-preview" />
```

Comment:
please update the new version in the sql.csproj too

Reply:
It is already updated